apollo-adminservice/src/main/scripts/startup.sh

```diff
@@ -63,6 +67,22 @@ function checkPidAlive {
     exit 1;
 }
 
+function existProcessUsePort() {
+    if [ "$(curl -X GET --silent --connect-timeout 1 --max-time 2 --head $SERVER_URL | grep "HTTP")" != "" ]; then
```

Comment:
I suppose this check won't work if the port doesn't expose HTTP protocol, can we use other commands like `lsof -i:$SERVER_PORT -sTCP:LISTEN -t`?

Reply:
Done

Reply:
There is another problem, i.e if port use by another linux user, this command won't work.

so, this port usage check should work when
* It doesn't expose HTTP protocol
* Port use by another linux user

maybe we can use some command like telnet?